server/config/urls.py

```diff
@@ -18,4 +19,5 @@
 
 urlpatterns = [
     url(r'^admin/', admin.site.urls),
+    url(r'^index/', TemplateView.as_view(template_name='index.html')),
```

Comment:
@ranmoshe 
please remove this line, no need to add a url only for this test, we can use an existing one.